amundsen_application/api/utils/notification_utils.py

```diff
@@ -1,5 +1,58 @@
-from flask import render_template
-from typing import Dict
+import logging
+
+from http import HTTPStatus
+
+from flask import current_app as app
+from flask import jsonify, make_response, render_template, Response
+from typing import Dict, List
+
+from amundsen_application.api.exceptions import MailClientNotImplemented
+
+
+def send_notification(notification_type: str, options: Dict, recipients: List, sender: str) -> Response:
```

Comment:
Let's use "keyword arguments" here, which is something that you will see in other places in our python code. The idea is to use explicit named arguments instead of relying on passing the arguments in the correct order.

This line would change to:
```
def send_notification(*, notification_type: str, options: Dict, recipients: List, sender: str) -> Response:
```
And the way it is called would change to:
```
send_notification(notifications_type=notification_type, options={}, recipients=[owner], sender=user.email)
```